client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -89,6 +91,16 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = () =>
 
 	const redirectToLatestStep = <Redirect to={ getLatestStepPath() } />;
 
+	function createSiteOrError() {
+		if ( newSiteError ) {
```

Comment:
`newSiteError` is still present when I return to the plans page from the error page.

That means if I navigate back using the browser button, I'll hit the error:

![Jun-15-2020 13-08-31](https://user-images.githubusercontent.com/6458278/84614685-3449b180-af0a-11ea-98e8-84e832bbca61.gif)

It should it the `redirectToLatestStep`.

Reply:
I'm unable to replicate in firefox, I do have an unresponsive back button though. I'll look at resetting the error state after routing to the error.

Reply:
Hmmm. I do not rule out that it's just me :) 7/10 it is.